lib/ovirt/service.rb

```diff
@@ -173,6 +173,7 @@ def probe_api_path(uri, path)
         request.get
       rescue RestClient::Exception => exception
         response = exception.response
+        raise exception if response.nil?
```

Comment:
A point of thought:
probe_api_path is called by find_api_path, which states in the comment: 
L187 # Probes all the candidate paths of the API, and returns the first that success. If all probes
L188 # fail, then the first candidate will be returned.
This will not be the new behaviour, now if it fails for any other reason on '/api' it will never try '/ovirt-engine/api', maybe the exception should be propagated from find_api_path only if both fail?